File: ClientSide/src/app/project/classes/trip.ts
import { Time } from "@angular/common";

export class trip{
    constructor(
        public tripCode?:number,
        public tripDestination?:String,
        public typeCode?:number,
        public tripDate?:Date,
        // public departureTime?:Time,
        public tripDurationHours?:number,
        public availablePlaces?:number,
        public price?:number,
        public photo?:String,
        public typeName?:String,
        public isFirstAid?:boolean
    )
    {}
}

File: ServerSide/Dal/Modules/Trip.cs
using System;
using System.Collections.Generic;

namespace Dal.Modules;

public partial 
    class Trip
{
    public int TripCode { get; set; }

    public string? TripDestination { get; set; }

    public int? TypeCode { get; set; }

    public DateTime? TripDate { get; set; }

    public TimeSpan? DepartureTime { get; set; }

    public int? TripDurationHours { get; set; }

    public int? AvailablePlaces { get; set; }

    public decimal? Price { get; set; }

    public string? Photo { get; set; }

    public virtual ICollection<BookingPlace> BookingPlaces { get; set; } = new List<BookingPlace>();

    public virtual TripType? TypeCodeNavigation { get; set; }
}
